packages/std/src/imports.rs

```diff
@@ -24,7 +24,7 @@ const HUMAN_ADDRESS_BUFFER_LENGTH: usize = 90;
 
 // This interface will compile into required Wasm imports.
 // A complete documentation those functions is available in the VM that provides them:
-// https://github.com/confio/cosmwasm/blob/0.7/lib/vm/src/instance.rs#L43
+// https://github.com/CosmWasm/cosmwasm/blob/v0.14.1/packages/vm/src/instance.rs#L53
```

Comment:
Thanks. The interesting code section moved down a bit and got much more advanced:

```suggestion
// https://github.com/CosmWasm/cosmwasm/blob/v0.14.1/packages/vm/src/instance.rs#L84-L201
```